examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 1789 (1.7K) [text/plain]\n",
+            "Saving to: ‘configs/quartznet_spkr_3x2x512_xvector.yaml’\n",
+            "\n",
+            "quartznet_spkr_3x2x 100%[===================>]   1.75K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2020-05-11 02:14:48 (28.4 MB/s) - ‘configs/quartznet_spkr_3x2x512_xvector.yaml’ saved [1789/1789]\n",
+            "\n",
+            "--2020-05-11 02:14:48--  https://raw.githubusercontent.com/NVIDIA/NeMo/master/scripts/scp_to_manifest.py\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 1742 (1.7K) [text/plain]\n",
+            "Saving to: ‘scp_to_manifest.py’\n",
+            "\n",
+            "scp_to_manifest.py  100%[===================>]   1.70K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2020-05-11 02:14:48 (42.4 MB/s) - ‘scp_to_manifest.py’ saved [1742/1742]\n",
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VgTR8CMlxu3p",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **SPEAKER RECOGNITION** \n",
+        "\n",
+        "Speaker Recognition (SR) is an broad research area which solves two major tasks: speaker identification (who is speaking?) and speaker verification (is the speaker who she claims to be?). In this work, we focmus on the far-field, text-independent speaker recognition when the identity of the speaker is based on how speech is spoken, not necessarily in what is being said. Typically such SR systems operate on unconstrained speech utterances, which are converted into vector of fixed length, called speaker embedding. Speaker embedding is also used in automatic speech recognition (ASR) and speech synthesis.\n",
+        "\n",
+        "As goal of most speaker related systems is to get good speaker level embeddings that could help distinguish from other speakers, we shall first train these embeddings in end-to-end manner optimizing the [QuatzNet](https://arxiv.org/abs/1910.10261) based encoder model on cross-entropy loss. We modify the original quartznet based decoder to get these fixed size embeddings irrespective of length of input audio. We employ mean and variance based statistics pooling method to grab these embeddings."
```

Comment:
As goal of most -> As the goal of most
irrespective of length of input audio. -> irrespective of the length of the input audio.

Reply:
Done